casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -29,192 +27,176 @@ class BlockQueryResponseAPITest
     with Inside
     with BlockDagStorageFixture
     with BlockApiFixture {
-  implicit val timeEff = new LogicalTime[Task]
-  implicit val spanEff = NoopSpan[Task]()
-  implicit val log     = Log.log[Task]
+  implicit val timeEff: LogicalTime[Task] = new LogicalTime[Task]
```

Comment:
We want to remove LogicalTime because it's wrong way to work with _virtual_ time in tests. So in all new code we want to avoid it, and here no virtual time is needed.
- https://github.com/rchain/rchain/issues/3730